hexpat/desc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<desc xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../_scripts/desc.xsd">
+   <targets>Java</targets>
```

Comment:
This grammar works for other targets, but not all because of "symbol conflicts." Please change the targets to `<targets>Antlr4ng;Cpp;CSharp;Go;Java;JavaScript;Python3;TypeScript</targets>`. It doesn't work for Dart, and the tester can't test Swift.